wordpress-mobile/gutenberg-mobile
# [iOS] Disable automatically managed signing for RN editor Xcode project.

### To test:

See the [Gutenberg PR](https://github.com/WordPress/gutenberg/pull/36181) for testing instructions.

### PR submission checklist:
